.github/workflows/build.yml

```diff
@@ -10,66 +10,70 @@ jobs:
-    - name: Check For setup.py
-      id: need-pypi
-      run: |
-        echo ::set-output name=setup-py::$( find . -wholename './setup.py' )
-    - name: Build Python package
-      if: contains(steps.need-pypi.outputs.setup-py, 'setup.py')
-      run: |
-        pip install --upgrade setuptools wheel twine readme_renderer testresources
-        python setup.py sdist
-        python setup.py bdist_wheel --universal
-        twine check dist/*
-    - name: Setup problem matchers
-      uses: adafruit/circuitpython-action-library-ci-problem-matchers@v1
+      - name: Dump GitHub context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: echo "$GITHUB_CONTEXT"
+      - name: Translate Repo Name For Build Tools filename_prefix
+        id: repo-name
+        run: |
+          echo ::set-output name=repo-name::$(
+          echo ${{ github.repository }} |
+          awk -F '\/' '{ print tolower($2) }' |
+          tr '_' '-'
+          )
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Versions
+        run: |
+          python3 --version
+      - name: Checkout Current Repo
+        uses: actions/checkout@v1
+        with:
+          submodules: true
+      - name: Checkout tools repo
+        uses: actions/checkout@v2
+        with:
+          repository: adafruit/actions-ci-circuitpython-libs
+          path: actions-ci
+      - name: Install dependencies
+        # (e.g. - apt-get: gettext, etc; pip: circuitpython-build-tools, requirements.txt; etc.)
+        run: |
+          source actions-ci/install.sh
+      - name: Pip install Sphinx, pre-commit
+        run: |
+          pip install --force-reinstall Sphinx sphinx-rtd-theme pre-commit
+      - name: Library version
+        run: git describe --dirty --always --tags
+      - name: Pre-commit hooks
+        run: |
+          pre-commit run --all-files
+      - name: Build assets
+        run: circuitpython-build-bundles --filename_prefix ${{ steps.repo-name.outputs.repo-name }} --library_location .
+      - name: Archive bundles
+        uses: actions/upload-artifact@v2
+        with:
+          name: bundles
+          path: ${{ github.workspace }}/bundles/
+      - name: Build docs
+        working-directory: docs
+        run: sphinx-build -E -W -b html . _build/html
+      - name: Check For setup.py
+        id: need-pypi
+        run: |
+          echo ::set-output name=setup-py::$( find . -wholename './setup.py' )
+      - name: Build Python package
+        if: contains(steps.need-pypi.outputs.setup-py, 'setup.py')
+        run: |
+          pip install --upgrade setuptools wheel twine readme_renderer testresources
+          python setup.py sdist
+          python setup.py bdist_wheel --universal
+          twine check dist/*
+      - name: Test Python package
+        run: |
+          pip install tox==3.24.5
+          tox
```

Comment:
These lines are the only parts of this file I meant to modify (add), but unless you use GitHub's "hide whitespace" option when looking at the diff, it looks like every line in the file was modified. My guess is the file may have been committed with Windows line endings previously, so when I saved the file in Linux all of the line endings got replaced.

Reply:
It's the indentation that have been changed.
Every line after `steps:` has 2 additional spaces.

Reply:
Ah good catch, I think VS Code must have done that automatically. I'll revert that.